ote/src/cljs/ote/views/service_search.cljs

```diff
@@ -92,11 +92,19 @@
         return-value (text/maybe-shorten-text-to 45 value-str)]
     return-value))
 
+(defn information-row
+  [title information]
+  [:div (stylefy/use-sub-style style/result-card :info-row)
```

Comment:
stylefy/use-sub-style style/result-card invoked three times, should you consider a helper function?